package/boot/arm-trusted-firmware-rockchip/Makefile

```diff
@@ -7,10 +7,10 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_VERSION:=2.10
+PKG_VERSION:=2.10.2
```

Comment:
unrelated change, this should be done in a separated PR.

Reply:
fixed